src/scenes/Dashboard/components/TopActions/index.js

```diff
@@ -14,10 +14,15 @@ import Card from 'material-ui/Card';
 import { withStyles } from 'material-ui/styles';
 
 import dashboardActions from '../../../../redux/Dashboard/actions';
+import CreateEvent from '../../../../scenes/CreateEvent/index';
 import { SortBy } from '../../../../constants';
 import styles from './styles';
 
 class TopActions extends Component {
+  state = {
+    createDialogOpen: false,
+  };
```

Comment:
does this work? Pretty sure this will throw an error with the `;` since  this is a class property.

Reply:
actually I believe it throws a lint error if you don't have the `;`. Even fat arrow funcs need the `;`.

Reply:
ditto deric

Reply:
weird... I'm surprised since you can't actually put logic inside a class, only methods inside a class. I don't have any lint errors for instance with:
```
state = {
  ...,
}
```